UI/data/locale/en-US.ini

```diff
@@ -462,6 +462,22 @@ MissingFiles.AutoSearchText="OBS has found additional matches for missing files
 MissingFiles.NoMissing.Title="Missing Files Check"
 MissingFiles.NoMissing.Text="No files appear to be missing."
 
+# macOS permissions dialog
+MacPermissions.Title="Review App Permissions"
+MacPermissions.Description="OBS Studio requires your permission to be able to provide certain features. It is recommended to enable these permissions, but they are not required to use the app. You can always enable them later via the OBS Studio menu."
+MacPermissions.AccessGranted="Access Granted"
+MacPermissions.RequestAccess="Request Access"
+MacPermissions.OpenPreferences="Open %1 Preferences"
+MacPermissions.Item.ScreenRecording="Screen Recording"
+MacPermissions.Item.ScreenRecording.Details="OBS requires this permission to be able to capture your screen."
+MacPermissions.Item.Camera="Camera"
+MacPermissions.Item.Camera.Details="OBS requires this permission to be able to capture content from a webcam or capture card."
+MacPermissions.Item.Microphone="Microphone"
+MacPermissions.Item.Microphone.Details="OBS requires this permission to be able to capture your microphone."
+MacPermissions.Item.Accessibility="Accessibility"
+MacPermissions.Item.Accessibility.Details="OBS requires this permission so that you can use keyboard shortcuts (hotkeys) while other apps are focused."
```

Comment:
The other Details strings use "to be able to". As that'd look strange here, we may have to tweak all the Details strings slightly to flow together better.

I think starting each with "OBS requires this permission" is a bit repetitive and may dissuade users from reading what's in the dialog.

Reply:
> we may have to tweak all the Details strings slightly to flow together better

Changed them a bit, but don't think they're great. I'm always open to suggestions (especially from native speakers).